packages/example-app-base/src/components/ApiReferencesTable/ApiReferencesTable.tsx

```diff
@@ -0,0 +1,497 @@
+          />
+        </Stack>
+        <Stack gap={SMALL_GAP_SIZE}>
+          <Text variant={'medium'}>Methods</Text>
+          <DetailsList
+            selectionMode={SelectionMode.none}
+            layoutMode={DetailsListLayoutMode.justified}
+            items={methods}
+            columns={this.methodColumns}
+            onRenderRow={this._onRenderRow}
+          />
+        </Stack>
+      </Stack>
+    ) : (
+      <Stack gap={SMALL_GAP_SIZE}>
+        <Text variant={'medium'}>Members</Text>
+        <DetailsList
+          selectionMode={SelectionMode.none}
+          layoutMode={DetailsListLayoutMode.justified}
+          items={properties}
+          columns={this.defaultColumns}
+          onRenderRow={this._onRenderRow}
+        />
+      </Stack>
+    );
+  }
+
+  private _renderExtends(): JSX.Element | undefined {
+    const { extendsTokens } = this.props;
+
+    return this._parseILinkTokens(true, extendsTokens);
+  }
+
+  private _renderDescription(): JSX.Element | undefined {
+    const { description } = this.props;
+
+    return description && description !== '' ? (
+      <Text variant={'medium'}>
+        <div dangerouslySetInnerHTML={{ __html: description }} />
+      </Text>
+    ) : (
+      undefined
+    );
+  }
+
+  private _renderTitle(): JSX.Element | undefined {
+    const { title, name } = this.props;
+
+    return title ? (
+      <Text variant={'xLarge'}>
+        <div dangerouslySetInnerHTML={{ __html: title }} id={name} />
+      </Text>
+    ) : (
+      undefined
+    );
+  }
+
+  private _onRenderRow(props: IDetailsRowProps, defaultRender?: IRenderFunction<IDetailsRowProps>): JSX.Element {
+    const { item } = props;
+    const rowStyles: Partial<IDetailsRowStyles> = {
+      root: {
+        selectors: {
+          ':hover': {
+            background: 'transparent'
+          }
+        }
+      },
+      isMultiline: {
+        wordBreak: 'break-word'
+      }
+    };
+
+    const deprecatedRowStyles = mergeStyles([
+      {
+        background: '#ffffcc'
+      },
+      'DetailsRow-deprecated'
+    ]);
+
+    if (item.deprecated === true) {
+      const deprecatedStyles: Partial<IDetailsRowStyles> = {
+        root: {
+          background: '#FFFFCC',
+          selectors: {
+            ':hover': {
+              background: '#FFFFCC'
+            }
+          }
+        }
+      };
+
+      return (
+        <div className={deprecatedRowStyles}>
+          Warning. This API is now obsolete.
+          {defaultRender!({
+            ...props,
+            styles: rowStyles && deprecatedStyles
+          })}
+        </div>
+      );
+    }
+
+    return <DetailsRow {...props} styles={rowStyles} />;
+  }
+
+  private _parseILinkTokens(extend: boolean, linkTokens?: ILinkToken[]): JSX.Element | undefined {
+    if (linkTokens && linkTokens.length > 0) {
+      if (extend) {
+        return (
+          <Text variant={'small'} className="ApiReferencesTable-extends">
```

Comment:
Where is this class name defined? Can we use some other approach instead?

Reply:
Good catch, it's not necessary anymore so I removed it!